grails-app/services/org/pih/warehouse/importer/InventoryLevelImportDataService.groovy

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2012 Partners In Health.  All rights reserved.
+ * The use and distribution terms for this software are covered by the
+ * Eclipse Public License 1.0 (http://opensource.org/licenses/eclipse-1.0.php)
+ * which can be found in the file epl-v10.html at the root of this distribution.
+ * By using this software in any fashion, you are agreeing to be bound by
+ * the terms of this license.
+ * You must not remove this notice, or any other, from this software.
+ **/
+package org.pih.warehouse.importer
+
+import grails.gorm.transactions.Transactional
+import org.pih.warehouse.data.DataService
+
+@Transactional
+class InventoryLevelImportDataService implements ImportDataService {
+    DataService dataService
+
+    void validateData(ImportDataCommand command) {
```

Comment:
let's add the `@Override` annotations if you implement methods from an interface. I know it's not necessary and the compiler understands it, but it's easier to read the code having those.